### ISSUE TITLE
clock module crashes on startup

### ISSUE THREAD

**Issue by jonorthwash** [apertium/phenny#491]
```
ERROR: Error during setup of clock.py module:
Traceback (most recent call last):
  File "/home/begiak/phenny/web.py", line 74, in wrapper
    return fn(doc, *args, **kw)
  File "/home/begiak/phenny/modules/clock.py", line 183, in scrape_wiki_time_zone_abbreviations
    name = cell.find('a').text
AttributeError: 'NoneType' object has no attribute 'text'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/begiak/phenny/bot.py", line 44, in module_control
    rephrase_errors(getattr(module, func), phenny)
  File "/home/begiak/phenny/tools.py", line 223, in rephrase_errors
    return fn(*args, **kw)
  File "/home/begiak/phenny/modules/clock.py", line 270, in setup
    scrape_wiki_zones(phenny)
  File "/home/begiak/phenny/modules/clock.py", line 246, in scrape_wiki_zones
    phenny.time_zone_abbreviations = scrape_wiki_time_zone_abbreviations()
  File "/home/begiak/phenny/web.py", line 78, in wrapper
    return fn(doc, *args, **kw)
  File "/home/begiak/phenny/modules/clock.py", line 183, in scrape_wiki_time_zone_abbreviations
    name = cell.find('a').text
AttributeError: 'NoneType' object has no attribute 'text'
```

**Comment by jonorthwash** [comment]
It appears that the scraper just needs to be updated.

**Comment by YashKarthik** [comment]
@jonorthwash  did it work after the update??

**Comment by jonorthwash** [comment]
@jonorthwash, no, I meant, the scraper needs to be updated (as in, fixed to deal with new format of data).  I didn't actually change anything.

**Comment by YashKarthik** [comment]
okok

**Comment by scoopgracie** [comment]
I claimed this on GCI. Will someone please assign it to me?

**Comment by jonorthwash** [comment]
This issue involves making the scraper work again and also not entirely crash if it encounters stuff it's not expecting (but it should definitely throw a warning, and maybe alert admins).

**Comment by Androbin** [comment]
Fixed by #503
